internal/mesh/internal/controllers/xds/controller_test.go

```diff
@@ -99,6 +103,9 @@ func (suite *xdsControllerTestSuite) SetupTest() {
 		leafCertEvents:   suite.leafCertEvents,
 		datacenter:       "dc1",
 	}
+
+	suite.isEnterprise = structs.NodeEnterpriseMetaInDefaultPartition().PartitionOrEmpty() == "default"
```

Comment:
Is `suite.isEnterprise` ever used? I wonder if we can remove this everywhere 🤔

Reply:
agreed